packages/static-wado-webserver/lib/routes/index.mjs

```diff
@@ -0,0 +1,10 @@
+import setStudyRoutes from "./study.mjs";
+
+/**
+ * Set all app routes.
+ *
+ * @param {*} router
+ */
+export default function setRoutes(router) {
```

Comment:
This part can have the conditional logic to add the various routes, but to do so it needs the configuration object currently being applied.